### ISSUE TITLE
yarn install fails with error on after lagoon v1.0.0 release

### ISSUE THREAD

**Issue by dasrecht** [uselagoon/lagoon#1208]
**Describe the bug**
After releasing of lagoon v1.0.0 we saw an error on running `yarn install` on certain projects

```
Step 7/15 : RUN yarn install --pure-lockfile
 ---> Running in d027d956a4fa
Error relocating /usr/bin/node: nghttp2_option_set_max_outbound_ack: symbol not found
ERROR: Service 'cli' failed to build: The command '/bin/sh -c yarn install --pure-lockfile' returned a non-zero code: 127
```

Running with the v0.24.0 images worked the latest images don't.

**To Reproduce**
Docker Testfile
```
FROM amazeeio/php:7.2-cli-drupal-v0.24.0 as builder-0240
RUN node --version
RUN yarn --version

FROM amazeeio/php:7.2-cli-drupal-v1.0.0 as builder-v100
RUN node --version
RUN yarn --version
```

Output:
```
Building cli
Step 1/6 : FROM amazeeio/php:7.2-cli-drupal-v0.24.0 as builder-0240
 ---> 1a967f2017c7
Step 2/6 : RUN node --version
 ---> Running in 37cf0676f341
v12.4.0
Removing intermediate container 37cf0676f341
 ---> f7489c1d3876
Step 3/6 : RUN yarn --version
 ---> Running in 210822809e03
1.16.0
Removing intermediate container 210822809e03
 ---> 01b1cffff91e

Step 4/6 : FROM amazeeio/php:7.2-cli-drupal-v1.0.0 as builder-v100
 ---> eff60a17cb35
Step 5/6 : RUN node --version
 ---> Running in 628aea3b42c3
Error relocating /usr/bin/node: nghttp2_option_set_max_outbound_ack: symbol not found
ERROR: Service 'cli' failed to build: The command '/bin/sh -c node --version' returned a non-zero code: 127
```

**Expected behavior**
Container build finishes without issues

**Workaround**
Downgrade to the latest v.0.24.0 image e.g `amazeeio/php:7.2-cli-drupal-v0.24.0`

**Additional context**
https://github.com/gliderlabs/docker-alpine/issues/342

**Comment by dasrecht** [comment]
Upon closer inspection i found that the library in question gets upgraded if when the pks are updated:

```
Building cli
Step 1/4 : FROM amazeeio/php:7.2-cli-drupal-v1.0.0 as builder-v100
 ---> eff60a17cb35
Step 2/4 : RUN apk upgrade
 ---> Running in 903652c8a4bd
fetch http://dl-cdn.alpinelinux.org/alpine/v3.10/main/x86_64/APKINDEX.tar.gz
fetch http://dl-cdn.alpinelinux.org/alpine/v3.10/community/x86_64/APKINDEX.tar.gz
(1/1) Upgrading nghttp2-libs (1.38.0-r0 -> 1.39.2-r0)
OK: 185 MiB in 92 packages
Removing intermediate container 903652c8a4bd
 ---> bf5163532e6b
Step 3/4 : RUN node --version
 ---> Running in f6063bb078d7
v12.9.0
Removing intermediate container f6063bb078d7
 ---> 59ef8f52fb19
Step 4/4 : RUN yarn --version
 ---> Running in f59d05acd9ef
1.17.3
Removing intermediate container f59d05acd9ef
 ---> b05ebc6fdd59

Successfully built b05ebc6fdd59
```

I'll give the cli containers a local rebuild to see if the newer version of `nghttp2-libs` gets installed

Reference: https://github.com/amazeeio/lagoon/blob/master/images/php/cli/Dockerfile#L27

**Comment by dasrecht** [comment]
Okay I dove a bit into how we build the php 7 images and found that we're using php:7-fpm-alpine
The php:7-fpm-alpine hasn't been updated with the latest version of of 1.39.2-r0 and holds the faulty 1.38.0-r0 which breaks node runs.

A workaround would be running

```
RUN apk upgrade
```
which updates the nghttp2-libs to the working version.

Reference: https://github.com/amazeeio/lagoon/commit/c1325504c92c31adbac7242db817f7444a264308

**Comment by AlexSkrypnyk** [comment]
@dasrecht Could you please confirm that `v1.0.1` has this fixed? I'm upgrading multiple projects to `v1.0.1` and they all fail with the same error as before the PR above. Adding `RUN apk upgrade` to the Docker image works, though. Can this command run in Lagoon?
